src/main.cpp

```diff
@@ -33,16 +33,18 @@ using namespace discovery_server;
 
 using namespace std;
 
-int main(int argc, char * argv[])
+int main(
+        int argc,
+        char* argv[])
 {
     // Initialize loging
     #if defined LOG_LEVEL_INFO
```

Comment:
```suggestion
#if defined LOG_LEVEL_INFO
```

Preprocessor directives are not indented. Please, apply to the other lines too.